### PR TITLE
Removed question marks from function names in some parts of the SDK

### PR DESCRIPTION
**Pull request by marcusnaslund**
Part of #1162 

@sebastianbaginski ?

### CODE REVIEW

**Comment by marcusnaslund** [comment]
Actually needs some more work.

**Comment by thomasfanell** [comment]
@marcusnaslund re-run PRInt, it should be cleared by magic now.

**Comment by marcusnaslund** [comment]
Yes, thank you.

**Comment by marcusnaslund** [comment]
Alright, ready for peer review. @sebastianbaginski

File: source/base/DateTime.ooc [hunk 1]
```diff
@@ -251,11 +251,11 @@ DateTime: cover {
 	}
 	/* validate argument ranges for hour/minutes/seconds vaules */
 	timeIsValid: static func (hour, minute, second, millisecond: Int) -> Bool {
-		hour in?(0 .. 24) && minute in?(0 .. 60) && second in?(0 .. 60) && millisecond in?(0 .. 1000)
+		hour in(0 .. 24) && minute in(0 .. 60) && second in(0 .. 60) && millisecond in(0 .. 1000)
 	}
 	/* validate argument ranges for year/month/day values */
 	dateIsValid: static func (year, month, day: Int) -> Bool {
-		year >= 1 && month in?(1 .. 13) && day in?(1 .. This daysInMonth(year, month) + 1)
+		year >= 1 && month in(1 .. 13) && day in(1 .. This daysInMonth(year, month) + 1)
```
Comment (sebastianbaginski) [inline source/base/DateTime.ooc]: `rock` won't confuse it with the existing `in` keyword ? I'm impressed

Comment (marcusnaslund) [inline source/base/DateTime.ooc]: When has `rock` ever let anyone down?

**Pull request merged**